bin/sl-deploy.js

```diff
@@ -56,16 +59,19 @@ while ((option = parser.getopt()) !== undefined) {
       local = true;
       break;
     default:
-      console.error('Invalid usage (near option \'%s\'), try `%s --help`.',
-        option.optopt,
-        $0);
+      console.error(GLB.t('msgInvalidUsage1', {
+        phOption: option.optopt,
+        phHelp: $0 + ' --help',
```

Comment:
In the string resource file, I have this: Invalid usage (near option `{phOption}`), try `{phHelp}`.

Alternative you mentioned would be this: Invalid usage (near option `{phOption}`), try {{`{phProgram} --help`}}.

Both yield the same result.  The former looks easier for human translators to understand the structure  The latter looks more complex, but more precise.  My angle is relatively more on the human translator side.

I thinks either way si fine.  Will change it as you mentioned in this case.